.appveyor.yml

```diff
@@ -28,6 +28,9 @@ install:
 test_script:
   - cd qcodes
   - python tests\dataset\legacy_DB_generation\generate_version_0.py
+  - python tests\dataset\legacy_DB_generation\generate_version_1.py
```

Comment:
Perhaps we don't need to keep the appveyor file up to date. I'd say it's safe to delete this file now.

Reply:
in another PR somewhen in the future in the galaxy far, or now?